### PR TITLE
Check if string is palindrome with stack 

### PR DESCRIPTION
**Pull request by reerajput930**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#4671

### CODE REVIEW

**Comment by reerajput930** [comment]
sir @HarshCasper, @siddharth25pandey, @bhav09, @ArchanGhosh,@ankitaggarwal23
I have made the PR,
my file can be tested

File: .deepsource.toml [hunk 1]
```diff
@@ -6,19 +6,14 @@ enabled = true
...
   [analyzers.meta]
-  import_paths = ["github.com/TesseractCoding/NeoAlgo"]
+  import_paths = ["github.com/reerajput930/NeoAlgo"]
 
 [[analyzers]]
 name = "javascript"
 enabled = true
-
-  [analyzers.meta]
-  environment = ["nodejs"]
```
Comment (HarshCasper) [inline .deepsource.toml]: Why are you making these changes?

File: C-Plus-Plus/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@
 # Codes in the C++ language
 
 ## Competitive Programming
-
+- [check if string is palindrome with stack](cp/check_palindrome.cpp)
```
Comment (HarshCasper) [inline C-Plus-Plus/README.md]: Write `Check if the String is a palindrome using Stack`

File: C-Plus-Plus/stack/check_palindrome.cpp [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+/*
+  sample input;
+  Enter the string: abbaa
+
+  sample output
+   0
+
+   ->TC =O(n);
+   ->SC =O(n);
+*/
```
Comment (HarshCasper) [inline C-Plus-Plus/stack/check_palindrome.cpp]: Add a new line at the EOF

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: .deepsource.toml (hunk 1 above)
Comment (reerajput930) [reply .deepsource.toml]: I Myself don't know, plz help.@harshcasper

**Comment by reerajput930** [comment]
@HarshCasper I have committed as per your requested changes.
@ankitaggarwal23  plz review it.

**Comment by HarshCasper** [comment]
> @HarshCasper I have committed as per your requested changes.
> @ankitaggarwal23 plz review it.

You have deleted the whole DeepSource TOML file. Please don't do that or else we would need to close the PR.

**Review by ankitaggarwal23 — COMMENTED** [review]
Are you still working on it.

**Review by ankitaggarwal23 — COMMENTED** [review]
Closing this due to inactivity since last 21 days. Thank you for your valuable time.😊

**Pull request closed without merging**